### PR TITLE
Implement Board

### PR DESCRIPTION
**Pull request by Fanarito**
Simple tested implementation of the Board class

### CODE REVIEW

File: src/main/hopur0/games/tictactoe/Board.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        }
+        return true;
+    }
+
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < state.length; i++) {
+            if (state[i] == null) {
+                sb.append('-');
```
Comment (dadisigursveinn) [inline src/main/hopur0/games/tictactoe/Board.java]: change from '-' to the number that this tile represents?

File: src/test/hopur0/games/tictactoe/BoardTest.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        b.setTile(p, 3);
+        b.setTile(p, 5);
+        b.setTile(p, 7);
+        b.setTile(p, 9);
+
+        assertEquals(b.full(), false);
+    }
+
+    @Test
+    public void testInvalidTile() {
```
Comment (dadisigursveinn) [inline src/test/hopur0/games/tictactoe/BoardTest.java:159]: no test for negative tiles

File: src/test/hopur0/games/tictactoe/BoardTest.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+            b.setTile(p, 1);
+            fail("Did not throw error");
+        }
+        catch (IllegalArgumentException ex) {
+            String expectedMessage = "Tile is already occupied";
+            assertEquals(expectedMessage, ex.getMessage());
+        }
+
+    }
+
```
Comment (dadisigursveinn) [inline src/test/hopur0/games/tictactoe/BoardTest.java:201]: No tests for the toString method

**Review by dadisigursveinn — CHANGES_REQUESTED** [review]

**Comment by Fanarito** [comment]
Fixed comments in newest commit

**Review by dadisigursveinn — APPROVED** [review]

**Pull request merged**